mobile/scc_poc_app/macos/firebase_app_id_file.json

```diff
@@ -0,0 +1,7 @@
+{
+  "file_generated_by": "FlutterFire CLI",
+  "purpose": "FirebaseAppID & ProjectID for this Firebase app in this directory",
+  "GOOGLE_APP_ID": "1:288316687003:ios:1db18e331c0d039fe93c3e",
```

Comment:
this is optional, if you think it is ok to expose app Id, and gcm sender id then we can leave it like this. If not, I would recommend to hide these as well, since this is a public repo.